patches/minecraft/net/minecraft/client/model/HumanoidModel.java.patch

```diff
@@ -0,0 +1,60 @@
+--- a/net/minecraft/client/model/HumanoidModel.java
++++ b/net/minecraft/client/model/HumanoidModel.java
+@@ -264,6 +_,8 @@
+          case TOOT_HORN:
+             this.f_102811_.f_104203_ = Mth.m_14036_(this.f_102808_.f_104203_, -1.2F, 1.2F) - 1.4835298F;
+             this.f_102811_.f_104204_ = this.f_102808_.f_104204_ - ((float)Math.PI / 6F);
++         default:
++            this.f_102816_.applyTransform(this, p_102876_, net.minecraft.world.entity.HumanoidArm.RIGHT);
+       }
+ 
+    }
+@@ -304,6 +_,8 @@
+          case TOOT_HORN:
+             this.f_102812_.f_104203_ = Mth.m_14036_(this.f_102808_.f_104203_, -1.2F, 1.2F) - 1.4835298F;
+             this.f_102812_.f_104204_ = this.f_102808_.f_104204_ + ((float)Math.PI / 6F);
++         default:
++            this.f_102815_.applyTransform(this, p_102879_, net.minecraft.world.entity.HumanoidArm.LEFT);
+       }
+ 
+    }
+@@ -396,7 +_,7 @@
+    }
+ 
+    @OnlyIn(Dist.CLIENT)
+-   public static enum ArmPose {
++   public static enum ArmPose implements net.minecraftforge.common.IExtensibleEnum {
+       EMPTY(false),
+       ITEM(false),
+       BLOCK(false),
+@@ -411,10 +_,30 @@
+ 
+       private ArmPose(boolean p_102896_) {
+          this.f_102890_ = p_102896_;
++         this.forgeArmPose = null;
+       }
+ 
+       public boolean m_102897_() {
+          return this.f_102890_;
+       }
++      // FORGE START
++      @javax.annotation.Nullable
++      private net.minecraftforge.client.IArmPoseTransformer forgeArmPose;
```

Comment:
This should be marked `final`, and correspondingly the patched-in constructor below should be modified to _not_ call the original constructor but set the `twoHandled` field itself.

Reply:
Ok, changed it now